src/mjolnir/statistics.h

```diff
@@ -10,8 +10,8 @@
 
 #include <sqlite3.h>
 
-#include "baldr/graphconstants.h"
 #include "midgard/aabb2.h"
+#include "valhalla/baldr/graphconstants.h"
```

Comment:
this one doesnt make sense to me. are you sure this diff is required?

Reply:
Yeah, you're right. I just tested it and it's fine without the change. It doesn't end up in the include directory in the installed location anyway, so I wonder how I tripped over it. I pushed a change.